pop/POPAnimator.mm

```diff
@@ -725,6 +770,21 @@ - (id)animationForObject:(id)obj key:(NSString *)key
   return animation;
 }
 
+- (CFTimeInterval)refreshPeriod
+{
+#if TARGET_OS_IPHONE
+  return self->_displayLink.duration;
+#else
+  if (NULL != self->_displayLink) {
+    CVTime period = CVDisplayLinkGetNominalOutputVideoRefreshPeriod(self->_displayLink);
+    if (period.flags & kCVTimeIsIndefinite)
+      return 0;
```

Comment:
In fact, this should never happen if POP had a proper display management. When the display becomes invalid, it should create a new valid display link or fallback to display timer. I intentionally fail here instead of returning an erroneous value.